server/server.js

```diff
@@ -293,6 +294,36 @@ app.get('/logout', (req, res) => {
     }
 });
 
+app.post('/flic_button_press', header(['button-serial-number','button-battery-level']).exists(), async (req, res) => {
+
+    try {
+        const validationErrors = validationResult(req)
```

Comment:
could do maybe

Reply:
separate task do you think @trisapeace ?

Reply:
the reason I used it here is because the information in these requests is all basically in the header, and the method we used looks only at the body of requests. express-validator I think handles both of these cases better and in a single module

Reply:
Totally separate task. I can look into it more during the brave-alert-lib thing.